### DISCUSSION TITLE
`<&` equivalent for Es

### DISCUSSION THREAD

**Discussion by asumbek** [wryun/es-shell#97] [Q&A] [answered]
Is there any equivalent for `<&` in Es? While `>&` have something similar which is `>[n=m]`, `<&` seems not.

I might be wrong since I read the manpage half-heartedly.

**Comment by jpco** [comment, marked as answer]
I might be missing something in your question, but `<[n=m]` is valid syntax (and parses exactly the same as `>[n=m]`, in fact)
```
; echo {foo >[1=2]}
{%dup 1 2 {foo}}
; echo {foo <[1=2]}
{%dup 1 2 {foo}}
```
Does the last line of the following example work how you expect?
```
; echo 'This is some file contents.' > my-file.txt
; cat my-file.txt 
This is some file contents.
; exec {<[3] my-file.txt}
; cat <[0=3]
This is some file contents.
```
(Note the last line works identically if written as `cat >[0=3]`.)

**Reply by asumbek** [reply]
That's what I wanted, thanks.
